test/TestIncludeMissFileWithRole.py

```diff
@@ -1,8 +1,7 @@
+import os
 from collections import namedtuple
 
 import pytest
-import os
-
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.